app/src/main/java/com/cornellappdev/android/eatery/MenuFragment.java

```diff
@@ -1,120 +1,111 @@
 package com.cornellappdev.android.eatery;
 
-import android.graphics.Color;
 import android.graphics.Typeface;
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.LinearLayout;
 import android.widget.TextView;
-import com.cornellappdev.android.eatery.Model.MealModel;
+
```

Comment:
Is there a reason for breaking up import statements?

Reply:
fixed!